CRM/EventsExtras/Test/Fabricator/Event.php

```diff
@@ -0,0 +1,46 @@
+<?php
+use CRM_EventsExtras_Test_Fabricator_Base as BaseFabricator;
+
+/**
+ * Class CRM_EventsExtras_Test_Fabricator_Event.
+ */
+class CRM_EventsExtras_Test_Fabricator_Event extends BaseFabricator {
+
+  /**
+   * Entity's name.
+   *
+   * @var string
+   */
+  protected static $entityName = 'Event';
+
+  /**
+   * Array if default parameters to be used to create an event.
+   *
+   * @var array
+   */
+  protected static $defaultParams = [
+    'title'  => 'Event Sample' ,
+    'event_type_id'  => 3 ,
+  ];
+
+  /**
+   * Fabricates an event  with the given parameters.
+   *
+   * @param array $params
+   *
+   * @return array
+   * @throws \CiviCRM_API3_Exception
+   */
+  public static function fabricate(array $params = []) {
+    // start_date should always be the future
```

Comment:
that's not true, the end date should always be in the future but not the start date, but the end date is not required anyway, so I will change this to : 

`$start_date = new DateTime();`